src/natcap/invest/coastal_vulnerability.py

```diff
@@ -724,7 +725,7 @@ def interpolate_shore_points(
     layer_name = os.path.splitext(
         os.path.basename(target_vector_path))[0]
     target_layer = target_vector.CreateLayer(
-        str(layer_name), aoi_spatial_reference, ogr.wkbPoint)

```

Comment:
As far as I can tell, `os.path.splitext` should always return a string. Was this left over from python 2 unicode vs. string?

Reply:
Yes, this looks like something we'd do on python2, where `layer_name` might be either `str` or `unicode` and `str` would always work in `CreateLayer`.  These days, `splitext` should only return text (to the best of my knowledge).